### ISSUE TITLE
Migrate from Jython 2.7 to Jython 3.7

### ISSUE THREAD

**Issue by slw07g** [NationalSecurityAgency/ghidra#194]
Support for Python 2 ends on January 1, 2020. Please migrate to a version of Jython that will be supported next year.

**Comment by slw07g** [comment]
Closed b/c there is no official Jython 3 version.

**Comment by dvdkon** [comment]
With Jython on life support it might make sense to use GraalVM's Python implementation. (Or it might not, depends on how many complex scripts would have to be ported). However, Jython seems to still be [somewhat maintained](https://bitbucket.org/jython/jython/commits/all) and the CPython EOL date won't affect it directly, so staying on the current Python version should be safe.

**Comment by wetneb** [comment]
See this issue about using GraalPython as a replacement for Jython: https://github.com/graalvm/graalpython/issues/96

tldr: running their Python implementation outside the GraalVM is discouraged by the maintainers.
